src/style.css

```diff
@@ -1,3 +1,82 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+::after,
+::before {
+  box-sizing: inherit;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  align-items: center;
+  margin: 10px 50px;
```

Comment:
This vertical 10px margin is not necessary. The header should be only 60px in height and it is like that without the header :)

Reply:
Great! Thank you for your advice!